Websites/webkit.org/wp-content/themes/webkit/standards-positions.php

```diff
@@ -0,0 +1,176 @@
+<script>
+(function () { // Begin closure
+
+function createElement(name, attributes = {}, child=null) {
+  const element = document.createElement(name);
+  for (const name in attributes) {
+    element.setAttribute(name, attributes[name]);
+  }
+  if (child !== null) {
+    element.append(child);
+  }
+  return element;
+}
+
+function addElement(parent, name, attributes = {}, child=null) {
+  return parent.appendChild(createElement(name, attributes, child));
+}
+
+function createStandardsPositionsTable() {
+  const fragment = new DocumentFragment();
+  const p = addElement(fragment, "p", { id: "filter-container" });
+  const input = addElement(p, "input", { class: "search-input", type:"search", id: "filter", placeholder: "Filter specifications…" });
+  input.oninput = updateStandardsPositionsTableRows;
+  const topicLabel = addElement(p, "label", {}, "By topic:");
+  const topicSelect = addElement(topicLabel, "select", { id: "filter-topic" });
+  topicSelect.oninput = updateStandardsPositionsTableRows;
+  const table = addElement(fragment, "table");
+  const headerRow = addElement(addElement(table, "thead"), "tr");
+  for (const heading of ["Issue", "Specification", "Concerns", "Position", "Topics", "Venues", "Self-link"]) {
+    addElement(headerRow, "th", { scope: "col" }, heading);
+  }
+  addElement(table, "tbody");
+  return fragment;
+}
+
+function createFilterTextRE(filterText) {
+  if (filterText) {
+    try {
+      return new RegExp(filterText, "i");
+    } catch(e) {}
+  }
+  return null; 
+}
+
+function createStandardsPositionsTableRows(issues, filterText = "", filterTopic = "") {
+  const fragment = new DocumentFragment();
+  const filterTextRE = createFilterTextRE(filterText);
+  for (const issue of issues) {
+    if (filterTextRE && issue["title"].search(filterTextRE) === -1) {
+      continue;
+    }
+    if (filterTopic && !issue["topics"].includes(filterTopic)) {
+      continue;
+    }
+    const id = issue["id"].substring("https://github.com/WebKit/standards-positions/issues/".length);
+    
+    const row = addElement(fragment, "tr", { id: `position-${id}` });
+    addElement(row, "td", {}, createElement("a", { href:issue["id"] }, `#${id}`));
+    if (issue["url"] !== null) {
+      addElement(row, "td", {}, createElement("a", { href:issue["url"] }, issue["title"]));
+    } else {
+      addElement(row, "td", {}, issue["title"]);
+    }
+    addElement(row, "td", {}, issue["concerns"].join(", "));
+    if (issue["position"] !== null) {
+      addElement(row, "td", { class: issue["position"].replace(" ", "-") }, createElement("span", {}, issue["position"]));  
+    } else {
+      addElement(row, "td");      
+    }
+    addElement(row, "td", {}, issue["topics"].join(", "));
+    addElement(row, "td", {}, issue["venues"].join(", "));
+    addElement(row, "td", {}, createElement("a", { href: `#position-${id}`, title: `Link to position: ${issue["title"]}` }, "⚓︎"));
+  }
+  return fragment;
+}
+
+let standardsPositions = null;
```

Comment:
This being used as a global is a bit confusing in the code... best to pass it as an argument. Similarly with others that are used in the same way.